### PR TITLE
Fixing the un-conventional highlight in dark mode in various sections of the websites

### PR DESCRIPTION
**Pull request by raj3000k**
## Description 📝

- Fixed the unconventional highlight in the dark mode in various sections of the websites(like Why Choose Us section on hovering).

Issue No #178 

https://github.com/PiyushKalyanpy/GyanaGuru/issues/178

## Checklist ✅

- [x] Reviewed the code changes thoroughly.
- [x] Tested the changes locally and ensured they work as expected.
- [x] Updated relevant documentation.
- [x] Added appropriate test cases, if applicable.
- [x] Considered potential security or performance implications.
- [x] Code adheres to the project's style guidelines.
- [x] Addressed all reviewer comments and concerns, if any.

# Before-
Screenshot 2023-05-26 at 12 14 40 PM

# After-

Screenshot 2023-05-30 at 2 12 52 PM

### CODE REVIEW

**Comment by raj3000k** [comment]
> > #178 Please provide a brief description of the purpose and goals of this pull request.
> > Include relevant screenshots or GIFs demonstrating the changes, especially for UI-related updates.
> 
> You need to remove these lines from the PR description.

Ok 👍🏻.

**Comment by raj3000k** [comment]
@lalitmee Done, Please check.

File: Components/WhyChooseUs.tsx [hunk 1]
```diff
@@ -33,11 +33,12 @@ const WhyChooseUs = () => {
 
 const InfoCard = ({ title, description }: any) => {
   return (
-    <div className=" p-4 hover:bg-seafoam transition-colors font-archivo rounded-3xl ">
-      {/* <span className=" absolute inline-flex -z-10 mt-2 h-4 w-20 rounded-full bg-seafoam opacity-75"></span> */}
+    <div className=" p-4 hover:bg-indigo-950 transition-colors font-archivo rounded-3xl ">
```
Comment (PiyushKalyanpy) [inline Components/WhyChooseUs.tsx:36]: instead of using a blue shade `hover:bg-indigo-950` please use green shade, so that I can match with the image.

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

File: Components/WhyChooseUs.tsx (hunk 1 above)
Comment (raj3000k) [reply Components/WhyChooseUs.tsx:36]: Like light green , or dark green or which?

**Review by PiyushKalyanpy — APPROVED** [review]

**Comment by PiyushKalyanpy** [comment]
🎉 Thank you @raj3000k for your valuable contribution to Gyanaguru Project. We would like to see more contributions from your side ✨

**Review by PiyushKalyanpy — APPROVED** [review]

**Pull request merged**